catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/federation/impl/SortedQueryMonitor.java

```diff
@@ -235,6 +236,7 @@ public void run() {
       returnProperties.put("hitsPerSource", hitsPerSource);
       returnProperties.put(
           ORIGINAL_SOURCE_PROPERTIES, (Serializable) Collections.unmodifiableMap(sourceProperties));
+      returnProperties.put("qm.sort.elapsed", System.nanoTime() - startTime);
```

Comment:
Either this key is misleading or it isn't measuring what you think it is measuring.  By using the start time when the run method is called and then calculating the elapsed time here, that calculates the total search time for all sources, including sorting/trimming as necessary.  The for loop above this is going through all of the sources in the query and waiting for them to be complete so that means their search time is included.  Sorting happens each time an individual source completes so there is really no way to capture just the "sort" time unless you are going to do that per source, in which case I would expect the name to be different and also calculated in a different spot. 

On the other hand, if this is meant to capture the full timing of the sorted query monitor for retrieving all of the source results, sorting them, and trimming to a page size then this is a fine calculation but as stated earlier, I think the name should be something different because just calling it "sort.elapsed" seems misleading if you don't know what it actually represents.